htlcswitch/switch.go

```diff
@@ -989,6 +976,7 @@ func (s *Switch) parseFailedPayment(deobfuscator ErrorDecrypter,
 	// the first hop. In this case, we'll report a permanent
 	// channel failure as this means us, or the remote party had to
 	// go on chain.
+	// TODO: check reason length instead
```

Comment:
what is meant by this/when do we plan to do this?

Reply:
Removed.